### DISCUSSION TITLE
<title>  Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.

### DISCUSSION THREAD

**Discussion by Raheemshah2809** [community/community#45841] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Not quite sure how to fix this, as I pay for it, and since starting at a new university, I can also get it for free, but it shows this message. 
<img width="914" alt="Screenshot 2023-01-30 at 11 03 59 pm" src="https://user-images.githubusercontent.com/66431658/215616467-ba9a8a97-71ce-4b8d-920d-63a073a4fa05.png">

If anyone knows how to fix this, any help would be appreciated. 
Thank you

**Comment by jge162** [comment]
If you are eligible for free plan and it’s not working. You should to submit a [support ticket](https://support.github.com/request/account?tags=hubberfy_account).
